lib/fusion_auth/openid_connect.ex

```diff
@@ -12,69 +12,69 @@ defmodule FusionAuth.OpenIdConnect do
   @type application_id :: String.t()
   @type tenant_id :: String.t()
   @type identity_provider :: %{
-    application_configuration: %{
-      application_id() => application_configuration()
-    },
-    button_image_url: String.t(),
-    button_text: String.t(),
-    debug: boolean(),
-    domains: [String.t()],
-    lambda_configuration_reconciled: String.t(),
-    linking_strategy: String.t(),
-    name: String.t(),
-    oauth_configuration: oauth_configuration(),
-    post_request: boolean(),
-    tenant_configuration_map: %{
-      tenant_id() => tenant_configuration()
-    },
-    type: String.t()
-  }
+          application_configuration: %{
```

Comment:
what formatter are you using just want to make sure its the same one. This file could have been generated without one previously.